creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -379,10 +433,11 @@ spec:
     pod: {}
 ----
 
-_______________________________________________________________________________________________
+______________________________________________________________________________
 *Note:* Masquerade is only allow to be connected to a pod network backend.
-*Note:* The network CIDR for can be configured in the pod network section using `vmNetworkCIDR`.
-_______________________________________________________________________________________________
+*Note:* The network CIDR for can be configured in the pod network section using
```

Comment:
CIDR for what?

Reply:
Don't make me fix all the typos and errors in the document pretty please. :)

Reply:
It was not malicious but I missed this fix from the upload. But now I am lazy enough to not bother and play the 'not my patch' card.